SU-SWS/stanford_profile
# D8CORE-941: WYSIWYG CKEditor styles

# READY FOR REVIEW
- Consolidate WYSIWYG paragraph styles in to one stylesheet
- Creates a CKEDitor stylesheet for wysiwyg styles.

# Summary
- Wysiwyg style stuff

# Needed By (Date)
- Sooner the better for content authors

# Urgency
- Before end of next sprint

# Steps to Test

1. Check out this branch
2. Clear all your caches
3. Edit the home page
4. Validate the styles are available in the CSS styles drop down
5. Validate the CSS stylesheets are loaded by creating a link in a heading within the WYSIWYG Editor and confirming it is red.

# Affected Projects or Products
- D8CORE

# Associated Issues and/or People
- D8CORE-941

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
